### DISCUSSION TITLE
Copilot generates syntactically incorrect code with VSCode quote matching

### DISCUSSION THREAD

**Discussion by rednebmas** [community/community#45487] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

See video. Does this happen to other people? 

It's not a big deal, just annoying. It does this 100% of the time.

https://user-images.githubusercontent.com/2336186/214983858-1d865c51-dfbd-4523-a478-f7d51b1b15f4.mov

**Comment by jge162** [comment]
Have you tried disabling/removing and reinstalling it?

**Comment by rednebmas** [comment]
@jge162 just tried removing and reinstalling and that didn't solve the issue

**Reply by jge162** [reply]
@rednebmas also you can reset copilot in settings. Have you done that yet? So it returns to like new status.

**Reply by rednebmas** [reply]
This @jge162? Didn't work.

 <img width="560" alt="Screenshot 2023-01-30 at 4 14 10 PM" src="https://user-images.githubusercontent.com/2336186/215617925-d7b5967b-a058-4467-9c5c-c17051afd2c4.png">

**Comment by stefaanMLB** [comment]
I'm having the same problem - still - surprised that not more people are bothered by this.
Is anybody using copilot for real ?  This really hurts productivity !
